README.md

```diff
@@ -1 +1,20 @@
-# hw01
\ No newline at end of file
+# Homework 1 
+
+![Me](https://tinyurl.com/y5ybnl62)
+
+## About Me
+
+### Blurb about Home
+I am originally from Miami, FL but, currently, my family lives in ~~south~~ southwest Florida. People who are from Miami proper insist that you refer to my part as **SOUTHWEST** Florida, rather than as *South* Florida because they somehow have sole ownership of the direction "south".
+
```

Comment:
It's really good to use comments!!

Reply:
Comments always help people to understand what you are doing. Also, good use of bold.